fonts/mscore/metadata.json

```diff
@@ -1,6 +1,6 @@
 {
     "fontName": "Emmentaler",
-    "fontVersion": 0.1,
+    "fontVersion": 2.002,
```

Comment:
wow, big step, is this normal?

Reply:
It never stepped. The font itself also has version numbers, which didn’t even follow normal font versioning schemes.

I’ve increased this to major version 2 to be sure it will register as newer than any old ones, and the minor 2 is from the F8vb/F15vb clef changes.